packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/BareManifest.java

```diff
@@ -57,7 +64,41 @@ public UpdateEntity getUpdateEntity() {
   }
 
   public ArrayList<AssetEntity> getAssetEntityList() {
-    // for now, don't copy any assets; this will be added in a later commit
-    return new ArrayList<>();
+    ArrayList<AssetEntity> assetList = new ArrayList<>();
+
+    AssetEntity bundleAssetEntity = new AssetEntity("bundle-" + mCommitTime.getTime(), "js");
+    bundleAssetEntity.isLaunchAsset = true;
+    bundleAssetEntity.embeddedAssetFilename = EmbeddedLoader.BARE_BUNDLE_FILENAME;
+    assetList.add(bundleAssetEntity);
+
+    if (mAssets != null && mAssets.length() > 0) {
+      for (int i = 0; i < mAssets.length(); i++) {
+        try {
+          JSONObject assetObject = mAssets.getJSONObject(i);
+          String type = assetObject.getString("type");
+          AssetEntity assetEntity = new AssetEntity(
+            assetObject.getString("packagerHash") + "." + type,
+            type
+          );
+          assetEntity.resourcesFilename = assetObject.optString("resourcesFilename");
+          assetEntity.resourcesFolder = assetObject.optString("resourcesFolder");
+
+          JSONArray scales = assetObject.optJSONArray("scales");
+          // if there's only one scale we don't to decide later on which one to copy
+          // so we avoid this work now
+          if (scales != null && scales.length() > 1) {
+            assetEntity.scale = (float)assetObject.optDouble("scale");
+            assetEntity.scales = new Float[scales.length()];
+            for (int j = 0; j < scales.length(); j++) {
+              assetEntity.scales[j] = (float)scales.getDouble(j);
+            }
```

Comment:
Might be prudent to sort the scales here to be extra robust.

Reply:
The way the `pickClosestScale` function is written now does not depend on the scales being sorted in any particular order.